package.json

```diff
@@ -7,6 +7,8 @@
     "prestart": "npm ci",
```

Comment:
I used `npm ci --production` recently as this is now also installing things from dev-dependencies. Especially mongo-memory-server is a big payload of 280MB which is useless to download. Can you add the --production flag?
Also shall we move nodemon to non-dev dependencies? This way people can use it if they cannot use pm2. Did I mention the flashing issue with pm2 did not occur on MTB3D's PC?

Reply:
Yes noticed myself. Will do

I can only gather it's the mis-match with pm2 local/global. It's the only thing I can put it down to at minute that I haven't personally tested.